bbb-learning-dashboard/src/components/QuizzesTable.jsx

```
@@ -318,40 +379,29 @@ const QuizzesTable = (props) => {
-          defaultMessage: 'Correct',
-        }) : intl.formatMessage({
-          id: 'app.learningDashboard.quizzes.incorrect',
-          defaultMessage: 'Incorrect',
-        });
+        const [, userAnswers] = params?.value;
+        return userAnswers || [];
       },
       renderCell: (params) => {
         let type = 'default';
-        const [isCorrect, responses] = params?.row[params?.field];
+        const [isCorrect] = params?.row[params?.field];
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)